.github/workflows/CI.yml

```diff
@@ -53,6 +56,13 @@ jobs:
         run: |
           cargo test --manifest-path ./client/Cargo.toml --no-default-features --features tokio --features openssl_crypto
 
+      - name: Test Server crypto/gnome (native)
```

Comment:
Test Server (native)

same below. and squash this commit with the first one.

Reply:
Done, thanks!